tsconfig.json

```diff
@@ -8,13 +8,13 @@
     "types": ["svelte", "vite/client"],
     "sourceMap": true,
     "baseUrl": "./",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
Just for my own understanding, why do we need to change this?

Reply:
Its not explicitly needed but I wanted to make the change since according to the typescript 5.0 article it's a good fit if we use Vite or a similar bundler

> If you are using a modern bundler like Vite, esbuild, swc, Webpack, Parcel, and others that implement a hybrid lookup strategy, the new bundler option should be a good fit for you.

Source: https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#moduleresolution-bundler